CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 - **[Feature]** Allow setting userId that applies to crashes, handled errors and push logs.
 - **[iOS][Breaking change]** Drop iOS 8 support. If you encounter errors during a build or the plugin installation please ensure your generated Podfile has proper target (i.e. `platform :ios, '9.0'`)
+- Added TypeScript Declarations for VS Code IntelliSense
+- Added daclarations for Analytics and crash methods.
```

Comment:
Lets merge this two both points to one if you don't mind
- Added declaration (d.ts) files